### DISCUSSION TITLE
How can I make the image fit the size of the Image Component with its rate of width and height constant?

### DISCUSSION THREAD

**Discussion by SanftMonster** [ant-design-blazor/ant-design-blazor#1641] [Q&A] [answered]
For example, in the following code
`<Image  Height="300px" Style="width:auto" Src="@item"></Image>`
I set the height of the component 300px.

However, the image is actually 640x480px. Then the image will be resized to 640x300 instead of 400x300 with its rate constant.

Is there an attribute I can set to resize the image with its rate of width and height constant?

If not, maybe I could try to implement it?

**Comment by ElderJames** [comment, marked as answer]
Thanks for contacting us @AsakusaRinne 
This is an issue about the style parameter, I will fix it later and ship it with 0.8.2

**Reply by ElderJames** [reply]
Fixed via #1642

**Reply by SanftMonster** [reply]
Got it! Thank you!
